code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/Identity.java

```diff
@@ -47,29 +46,27 @@ private Identity() {}
 	 *
 	 * @return The version as {@code String}
 	 */
+	@NonNull
 	public static String extensionVersion() {
 		return IdentityConstants.EXTENSION_VERSION;
 	}
 
 	/**
 	 * Registers the extension with the Mobile SDK. This method should be called only once in your application class.
 	 *
-	 * @deprecated Use {@link MobileCore#registerExtensions(List, AdobeCallback)} with {@link Identity#EXTENSION} instead.
+	 * @deprecated as of 2.0.0, use {@link MobileCore#registerExtensions(List, AdobeCallback)} with {@link Identity#EXTENSION} instead.
 	 */
 	@Deprecated
+	@SuppressWarnings("deprecation")
```

Comment:
Why are we suppressing warnings here?

Reply:
it shows a warning in the IDE when using the registerExtension API so it suppresses that, but you would still see the deprecation message on the API itself